extensions/wikia/AchievementsII/css/oasis.scss

```diff
@@ -190,7 +180,7 @@
 		}
 	}
 	
-	& > div.UserProfileAchievementsModule {
+	& .UserProfileAchievementsModule {
```

Comment:
Yeah, there is probably no need for ampersand at the beginning anymore, since we're nesting this class normally, not targetting the exact children (one-level descendants).

Unless you wanted to target combined classnames, then there is no need for the space character.